bower.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "cfgov-sheer-templates",
   "description": "Sheer templates for consumerfinance.gov",
-  "version": "0.3.1",
+  "version": "1.0.0",
```

Comment:
That decision happened before you started contributing, right? I don't understand the logic behind that. I think it's valuable to have it available in the code.